benchmark/swebench-rna-first/issue827/rna_traverse.py

```
@@ -0,0 +1,690 @@
+        "--repo",
+        CONFIG["repo"],
+        "--root",
+        CONFIG["root"],
+        "--node",
+        args.node,
+        "--mode",
+        args.mode,
+        "--compact",
+    ]
+    started = time.monotonic()
+    result = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)